precompile/erc20/allowance.go

```diff
@@ -19,37 +18,40 @@ const (
 	AllowanceMethodName = "allowance"
 )
 
-type allowanceMethod struct {
+type AllowanceMethod struct {
 	authzkeeper authzkeeper.Keeper
```

Comment:
In most of the precompiled methods such as `approve` there was a change to make a struct upper case to make it public. This one is private. Is this intentional?

Reply:
Not sure if I get it. `AllowanceMethod` is public.

Reply:
🤦 I looked at the wrong place, nvm.